DX-1274: Features refactoring (#749)
SwedbankPay/developer.swedbankpay.com: payment-instruments/card/features/optional-features/index.md

---
title: Optional features
redirect_from:
estimated_read: -
description: |

>   Welcome to Optional Features - a subsection of Features.
>   This section has extented code examples and features that were not
>   covered by the other subsections.

Reviewer: I'm not happy about my suggested description, but I think we can be a bit terser and on point. We should also have a card here for each optional feature, no?

```suggestion
  This section details the features of Card that are optional.
```
